### PR TITLE
Assignments Endpoints and Admin Batch Sign Up Endpoint

### PR DESCRIPTION
**Pull request by adityavinodk**
Added endpoints for Assignments and endpoint for admin batch signup which can be accessed by localhost:8080/admin

### CODE REVIEW

**Comment by aniketnk** [comment]
Yo, separate PRs for 2 different features !

File: server/routes/api/assignments.js [hunk 1]
```diff
@@ -0,0 +1,403 @@
...
+            }, {
+                $push:{
+                    submissions: {
+                        'user':req.params.userID,
+                        'file':req.file._id
+                    }
+                }
+            }, {new:true}, function(err, assignment){
+                if(err){
+                    console.log("aditya");
```
Comment (aniketnk) [inline server/routes/api/assignments.js]: I am aniket

File: server/routes/api/assignments.js [hunk 2]
```diff
@@ -0,0 +1,403 @@
...
+                success: false,
+                message: "Error: Enter assignment ID"
+            });
+        }
+        Course.find({
+            students: req.params.userID,
+            assignments: req.params.assignmentID
+        }, function(err, courses){
+            if(err){
+                console.log("hello");
```
Comment (aniketnk) [inline server/routes/api/assignments.js]: HIIII, what up!

**Comment by adityavinodk** [comment]
Please check the Upload Assignments endpoint

**Pull request merged**